### PR TITLE
Vonage.VonageBusiness version 2.11.0

### PR DESCRIPTION
**Pull request by ghost**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/64634)

### CODE REVIEW

File: manifests/v/Vonage/VonageBusiness/2.11.0/Vonage.VonageBusiness.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Vonage.VonageBusiness
+PackageVersion: 2.11.0
+PackageLocale: en-US
+Publisher: Vonage
+PackageName: Vonage Business
+License: Vonage License
```
Comment (ItzLevvie) [inline manifests/v/Vonage/VonageBusiness/2.11.0/Vonage.VonageBusiness.locale.en-US.yaml:9]:
```suggestion
License: Proprietary
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Comment by Madhusudhan-MSFT** [comment]
@davidford88 - I am looking to this PR and could you help me finding download link from the official product site ? I see installer in this PR hosted on storage.googleapis.com domain, but would like if it any different installer than what we see in the official product site.

File: manifests/v/Vonage/VonageBusiness/2.11.0/Vonage.VonageBusiness.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Vonage.VonageBusiness
+PackageVersion: 2.11.0
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
+  InstallerUrl: https://storage.googleapis.com/ele-it-jumpcloud/VonageBusinessSetup.exe
```
Comment (ItzLevvie) [inline manifests/v/Vonage/VonageBusiness/2.11.0/Vonage.VonageBusiness.installer.yaml:9]:
```suggestion
  InstallerUrl: https://vbc-downloads.vonage.com/win/VonageBusinessSetup.exe
```
from https://www.vonage.com/downloads/

InstallerSha256 is the same (ed7c59c79dfaf31f3258d127325d15e9db6c8c9f795299c41e7d56f2da768b53)

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request closed without merging**